src/createServer.js

```diff
@@ -1,8 +1,66 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const fileName = normalizedURL.pathname.slice(1);
+
+    res.setHeader('Content-Type', 'text/plain');
+
+    if (!req.url.startsWith('/file')) {
+      res.statusCode = 400;
+      res.end('Invalid request');
+
+      return;
+    }
```

Comment:
The check `if (!req.url.startsWith('/file'))` is redundant because the subsequent check `if (!req.url.startsWith('/file/'))` already covers this case. You can remove the first check to simplify the code.